cmd/earth/main.go

```diff
@@ -184,9 +184,9 @@ func newEarthApp(ctx context.Context, console conslogging.ConsoleLogger) *earthA
 		},
 		&cli.StringFlag{
 			Name:        "git-url-instead-of",
-			Value:       "git@github.com:=https://github.com/",
+			Value:       "https://gitlab.com/=git@gitlab.com:,https://github.com/=git@github.com:",
```

Comment:
I'm thinking we should change the type of this flag to a `StringSliceFlag`. It feels more natural for the user to provide multiple values as

```bash
earth --git-url-instead-of 'https://gitlab.com/=git@gitlab.com:' \
  --git-url-instead-of 'https://github.com/=git@github.com:' \
  ...
```

rather than

```bash
earth --git-url-instead-of 'https://gitlab.com/=git@gitlab.com:,https://github.com/=git@github.com:' ...
```

What do you think?

(Side note: In the future, we should separate global config like this one to its own config file. But that is a matter for another day)

Reply:
FTR We decided to leave this off for another PR.